src/ralph/attachments/models.py

```diff
@@ -117,14 +121,30 @@ def __str__(self):
             self.original_filename, self.mime_type, self.uploaded_by
         )
 
+    def _md5_from_file(self):
```

Comment:
`_md5_from_file suggest` "file" argument, so maybe change it to the 

`get_md5_sum()`